rx/core/operators/groupbyuntil.py

```diff
@@ -37,6 +41,9 @@ def _group_by_until(key_mapper: Mapper,
 
     element_mapper = element_mapper or identity
 
+    default_subject_mapper = lambda: Subject()
+    subject_mapper = subject_mapper or default_subject_mapper
```

Comment:
simply use Subject instead of a lambda ?
```
subject_mapper = subject_mapper or Subject
```

Reply:
thanks :+1: